### PR TITLE
Send reminder

### PR DESCRIPTION
**Pull request by rjmk**
Working on the dry run ui

### CODE REVIEW

**Comment by rjmk** [comment]
Might not merge yet

File: api/controllers/PrivateController.js [hunk 1]
```diff
@@ -47,23 +48,40 @@ module.exports = {
...
   showMaintenance: function (req, res) {
     res.view('pages/maintenance', {user: req.session.user})
   },
   Upload: function (req, res) {
     /**
-     *	The sign '&' (ampersand) splits the
-     *	request body content in different objects
-     *	where all the keys are the data.
+     *  The sign '&' (ampersand) splits the
+     *  request body content in different objects
```
Comment (FilWisher) [inline api/controllers/PrivateController.js:74]: nice indenting

Comment (rjmk) [inline api/controllers/PrivateController.js:74]: worth a diff

Comment (FilWisher) [inline api/controllers/PrivateController.js:74]: should it go in one more time?

Comment (rjmk) [inline api/controllers/PrivateController.js:74]: Obviously that would be preferable, but I don't know if we have sufficient hard disk space for that

Comment (rjmk) [inline api/controllers/PrivateController.js:74]: (This is the cruel hand of `:retab` by the way)

Comment (FilWisher) [inline api/controllers/PrivateController.js:74]: [my old friend](https://github.com/FilWisher/c-crisp/commit/f959f74a174cc8a9277d69a3cf6b5963b64ce712)

**Pull request merged**